tests/integration/integration_nodejs_test.go

```diff
@@ -601,3 +601,59 @@ func TestLargeResourceNode(t *testing.T) {
 		Dependencies: []string{"@pulumi/pulumi"},
 	})
 }
+
+func testComponentPathEnv() (string, error) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+	absCwd, err := filepath.Abs(cwd)
+	if err != nil {
+		return "", err
+	}
+	pluginDir := filepath.Join(absCwd, "construct_component", "testcomponent")
+	return "PATH=" + os.Getenv("PATH") + ":" + pluginDir, nil
+}
+
+// Test remote component construction in Node.
+func TestConstructNode(t *testing.T) {
```

Comment:
Do we need to skip this test on Windows since `pulumi-resource-testcomponent` is a shell script? Or should we make the test emit an appropriate `pulumi-resource-testcomponent` script depending on if it's Unix or Windows?

Reply:
> Or should we make the test emit an appropriate pulumi-resource-testcomponent script depending on if it's Unix or Windows?

I'll check in a pulumi-resource-testcomponent batch script.